### PR TITLE
Updated to include icons on the buttons

### PR DESCRIPTION
**Pull request by jwiese-ms-zz**
Added code to put icons on the blue controller buttons to make it more obvious what the buttons do.

### CODE REVIEW

**Comment by jwiese-ms-zz** [comment]
One minor change to have consistent look and feel of the buttons, change line 286:

widget.typeOfButton ==ButtonType.Down ? Icons.**keyboard_arrow_down**

**Comment by boyan01** [comment]
thanks for your contribution first, but it seems that this feature has a simpler implementation.

maybe we do not need the enum class `ButtonType`

we can simplly pass a Widget type param to _Button constructor method like this

```dart
_Button(
    size: _DIR***SIZE,
    icon: Icon(Icons.***), // set button icon directly
    onTap: () {
         //do some work
    }
)
```
this implement might be more consice , i think.

thanks for your contribution again.

**Comment by jwiese-ms-zz** [comment]
true, that would make it a little more concise.  If the _Button ctor takes an Icon you could simply then in the build for _ButtonState reference it there.

**Comment by jwiese-ms-zz** [comment]
ok, so this works but the one problem is the rotating of the images. I hadn't dug into it but it appears you've rotated them at some point because if I just put the image in directly they are rotated…thus my transform.  The problem is that the big button is not rotated.  So, either the small ones need to be "un rotated", the big one needs to be "rotated the same as the small" or you do what I did and transform only the icons on the small buttons.

File: lib/panel/controller.dart [hunk 1]
```diff
@@ -181,11 +181,9 @@ class _SystemButton extends StatelessWidget {
   }
 }
 
-enum ButtonType { MoveLeft, RotateRight, MoveRight, Down, Drop }
-
 class _Button extends StatefulWidget {
   final Size size;
-  final ButtonType typeOfButton;
+  final IconData buttonIcon;
```
Comment (boyan01) [inline lib/panel/controller.dart:186]: the `buttonIcon` can be a `Widget` type. then the rotation problem could be easy to fix.
```dart
Button(
        enableLongPress: false,
        size: Size(90, 90),
        icon: Transform.rotate(angle: - (math.pi/ 12.0) * 3,child: Icon(Icons.file_download)),
        onTap: () {
          Game.of(context).drop();
        });
```

**Review by boyan01 — APPROVED** [review]
And the file_download icon looks so strange  on Drop Down Button, maybe remove the icon for Drop Down is the best practice.

ok, i'll take over this feature , try to add icon and description for each button

**Pull request merged**